### ISSUE TITLE
[Spell] Polymorph randomly breaking

### ISSUE THREAD

**Issue by Eternae1** [SunwellWoW/Sunwell-TBC-Bugtracker#776]
Description: Polymorph spell is randomly breaking on some mobs, I have only noticed it break on mobs that used to be or should be elite (such as the blackhand orcs in eastern Redridge)

How it works: Cast Polymorph on a mob and they break out within 3 or 4 seconds. (No damage or modifiers done to the mob)

How it should work: Cast Polymorph on a mob and they should stay in it for the full duration. I believe elites have a chance to break out early, but not always within first few seconds.

Source (you should point out proofs of your report, please give us some source):

**Comment by notsofamous** [comment]
All CC is affected by heartbeat resists, meaning that it can break at any point. Is it consistently breaking very early?

**Comment by Eternae1** [comment]
Yes, but only on mobs that used to be elite or are elite but missing the nameplates. non-elite beasts and humanoids are fine.

**Comment by arcwizard** [comment]
I also noticed such a bug.  Look screenshot, after 8 seconds the mob comes out of the polymorph.
![wowscrnshot_102518_170812](https://user-images.githubusercontent.com/43764742/47507193-04035a00-d872-11e8-9b86-2c16701f827a.jpg)

**Comment by akrom23** [comment]
I believe it must have been a heartbeat resist. On 29 level, your Poly have 30 sec duration. After 4 seconds, a heartbeat resist may happen.
